### PR TITLE
Update Switch.tsx

### PR DESCRIPTION
**Pull request by ryanyae**
## Description
This is the WIP ticket, conditionally rendering components on the website to be seen in development and not in production. 
MUST ADD: **REACT_APP_WIP=true** in your .env file that is in root folder
## Updates

[Please enter a brief summary of updates to this PR during code review - if any]

## Checklist

- [ ] Ran `npm run lint:fix` to format code
- [ ] Requested at least one reviewer
- [ ] Connected PR to Trello ticket (steps below)
- [ ] Addressed code review comments
- [ ] Gained at least one approval for your PR

## Connecting your PR to the corresponding ticket:
- Click on the "GitHub" button found on the right of your Trello ticket
- Choose "Attach PR" from the dropdown list
- Link your GitHub account (if you haven't done so already)
- Navigate to`VCL-content-platform` and choose your PR from the dropdown.

### CODE REVIEW

**Comment by ElisaHu1** [comment]
No I still cannot see the projects 
image

**Comment by ryanyae** [comment]
@ElisaHu1 That is so weird, I wonder why this is happening

**Comment by ryanyae** [comment]
@ax920 I was wondering if the problem that Elisa is encountering is persistent on your side as well?

**Comment by ax920** [comment]
> @ax920 I was wondering if the problem that Elisa is encountering is persistent on your side as well?

I'm getting this problem as well. Is it up to date (rebased onto main)?

**Comment by ryanyae** [comment]
@ax920 @ElisaHu1 I just pushed a recent commit that should have everything working properly. I also changed the value of the .env variable, which is now REACT_APP_WIP=developement if you want to see development-view and REACT_APP_WIP=production if you would like to see the production-view.

File: src/components/Navbar/Navbar.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ import AccountCircleIcon from "@mui/icons-material/AccountCircle";
 import MobileMenu from "@components/MobileNavbar";
 require("dotenv").config();
 
-const IS_WIP = process.env.REACT_APP_WIP;
+const IS_WIP = process.env.REACT_APP_ === 'developement';
```
Comment (ax920) [inline src/components/Navbar/Navbar.tsx:26]: Nit: development

Comment (ax920) [reply src/components/Navbar/Navbar.tsx:26]: Applies elsewhere as well

Comment (ryanyae) [reply src/components/Navbar/Navbar.tsx:26]: 😂😂😂😂😂😂😂. I will make sure to get this fixed ASAP

**Review by ElisaHu1 — APPROVED** [review]

**Pull request merged**